src/biogeochem/VOCEmissionMod.F90

```diff
@@ -84,14 +84,17 @@ module VOCEmissionMod
   !------------------------------------------------------------------------
   subroutine Init(this, bounds)
 
-    use clm_varctl     , only : use_fates, use_fates_sp
+    use clm_varctl     , only : use_fates, use_fates_sp, use_fates_nocomp
```

Comment:
per above, we might not need the use_fates_sp flag at all?

Reply:
Sounds good! That's a great simplification.